Makefile

```diff
@@ -217,7 +217,10 @@ clean:
 	rm -rf build/_output/bin/ptp-operator
 
 functests:
-	hack/run-functests.sh
+	SUITE=./test hack/run-functests.sh
```

Comment:
please move the older tests to conformance

like

test/
-> conformance
--> ptp

Reply:
done